Cargo.toml

```diff
@@ -21,5 +21,15 @@ features = ["derive"]
 
 [dev-dependencies]
 env_logger = "0.7"
-winit = "0.24"
+winit = { version = "0.24", features = ["web-sys"] }
 futures = "0.3"
+
+[target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies]
+pollster = "0.2"
+wgpu-subscriber = "0.1"
```

Comment:
What does `wgpu-subscriber` do exactly? Does it replace `env_logger`?

Reply:
Yes and it's a pretty logger for web env, and more for Chrome as some code is dedicated to that browser.
In Chrome you can select log level, that `wgpu-subscriber` detect and print the right log level.